pkg/innerring/internal/blockchain/items.go

```diff
@@ -0,0 +1,47 @@
+package blockchain
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+)
+
+// TODO: think how to merge functions with ones from client package (like client.ArrayFromStackItem)
```

Comment:
There is an `unwrap` for it. Current code can use anything it has already, but long-term it's `unwrap` for generic items + custom (per-contract probably) functions for custom things (like some structures).